selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java

```diff
@@ -76,22 +76,36 @@ public CommandsEditor(
     PageFactory.initElements(seleniumWebDriver, this);
   }
 
-  public static final class CommandsEditorType {
-    public static final String COMMAND_LINE_EDITOR =
-        "(" + Locators.ORION_ACTIVE_EDITOR_CONTAINER_XPATH + ")[1]";
-    public static final String PREVIEW_URL_EDITOR =
-        "(" + Locators.ORION_ACTIVE_EDITOR_CONTAINER_XPATH + ")[2]";
+  public enum CommandsEditorType {
+    COMMAND_LINE_EDITOR("(" + ORION_ACTIVE_EDITOR_CONTAINER_XPATH + ")[1]"),
+    PREVIEW_URL_EDITOR("(" + ORION_ACTIVE_EDITOR_CONTAINER_XPATH + ")[2]");
 
-    private CommandsEditorType() {}
+    private final String locator;
+
+    CommandsEditorType(String locator) {
+      this.locator = locator;
+    }
+
+    public String get() {
+      return this.locator;
+    }
   }
 
-  public static final class CommandsMacrosLinkType {
-    public static final String EDITOR_MACROS_LINK =
-        "(" + ACTIVE_EDITOR_ENTRY_POINT + "//a[@id='gwt-debug-link-explore_macros'])[1]";
-    public static final String PREVIEW_MACROS_LINK =
-        "(" + ACTIVE_EDITOR_ENTRY_POINT + "//a[@id='gwt-debug-link-explore_macros'])[2]";
+  public enum CommandsMacrosLinkType {
```

Comment:
IMHO this enum describes locators, so the name `CommandsMacrosLinkLocator` is more precise, as for me.